plugin-server/src/worker/ingestion/action-manager.ts

```diff
@@ -168,3 +210,18 @@ export async function fetchAction(client: PostgresRouter, id: Action['id']): Pro
     const action: Action = { ...rawActions[0], steps: rawActions[0].steps_json ?? [], hooks }
     return action.post_to_slack || action.hooks.length > 0 ? action : null
 }
+
+export async function fetchPluginConfigsWithMatchActions(
+    postgres: PostgresRouter
+): Promise<Pick<PluginConfig, 'id' | 'team_id' | 'match_action_id'>[]> {
+    const { rows }: { rows: Pick<PluginConfig, 'id' | 'team_id' | 'match_action_id'>[] } = await postgres.query(
+        PostgresUse.COMMON_READ,
+        `SELECT posthog_pluginconfig.id, posthog_pluginconfig.team_id, posthog_pluginconfig.match_action_id 
+            FROM posthog_pluginconfig
+            WHERE posthog_pluginconfig.match_action_id IS NOT NULL 
+            AND posthog_pluginconfig.enabled`,
```

Comment:
this would be more readable
```suggestion
        `SELECT id, team_id, match_action_id 
            FROM posthog_pluginconfig
            WHERE match_action_id IS NOT NULL 
            AND enabled`,
```

Reply:
Mostly copied from the other functions but sure 👍